content/3_Silver/Intro_Bitwise.mdx

```diff
@@ -406,6 +406,41 @@ int main() {
 ```
 
 </CPPSection>
+
+<PySection>
+
+```py
+def main():
+	import sys
+
+	input = sys.stdin.read
```

Comment:
is a main function and sys.stdin necessary

Reply:
No, a main method is not necessary for this problem, so I removed it. Also, I replaced the sys.stdin with regular input.
Here is the AC screenshot for the new changes:

![image](https://github.com/user-attachments/assets/4bb54a77-50ce-403c-b6d4-992ef03ad1a9)